src/components/Question.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import {
+  Dimensions,
+  View
+} from 'react-native';
+import EStyleSheet from 'react-native-extended-stylesheet'
+import StyledMarkdown from './StyledMarkdown'
+import { connect } from 'react-redux'
+import { setState } from '../actions/index'
+
+//questionContainerHeight is stored in redux because the swipeable component needs
+//to be absolutely positioned outside of the ContainerPanel for Android
+//This is to help calculate that height
+
+const mapStateToProps = (state, ownProps) => ({
+  questionContainerHeight: state.classifier.questionContainerHeight[ownProps.workflowID] || 0,
+})
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  setQuestionContainerHeight(height) {
+    dispatch(setState(`classifier.questionContainerHeight.${ownProps.workflowID}`, height))
+  },
+})
+
+export class Question extends Component {
+  render() {
+    return (
+      <View style={[styles.questionContainer, { height: this.props.questionContainerHeight }]}>
+        <View style={styles.question}>
+          <StyledMarkdown
+            extraCSS={ 'p {font-size: 14px; font-weight: 500; margin: 5px 0 0;}' }
+            markdown={ this.props.question }
+            width={ Dimensions.get('window').width - 100 }
+            onResize={ (newHeight) => this.props.setQuestionContainerHeight(newHeight) }
```

Comment:
Yeah, this is strange.  It turns out that when you use webview you have to hardcode the height of the webview container.  Unfortunately for most everything with markdown we won't know the height until rendered (for example the Question could be one, or a few lines long).  As the result of this, there's a workaround with an event listener on the markdown html template that sets the size, and this is the callback from that.  I'll rename the function though, since by the name `onResize` it's not very obvious that this is what it's doing.